scene/gui/spin_box.cpp

```diff
@@ -294,6 +297,14 @@ void SpinBox::apply() {
 	_text_submitted(line_edit->get_text());
 }
 
+void SpinBox::set_custom_arrow_step(double p_custom_arrow_step) {
+	custom_arrow_step = p_custom_arrow_step;
+}
+
+double SpinBox::get_custom_arrow_step() const {
+	return custom_arrow_step;
+}
```

Comment:
These should also be renamed to remove the `custom_` prefix I think (including the member var). Remember to update the binding and the docs too.

Reply:
What do you mean remove? Isn't the point of the changes requested by juan that we ADD the `custom_` prefix?

Reply:
Ah yes. Then you just forgot to update the property name :)

Reply:
Ah I see, yeah that sounds likely, I will fix it now.